sdks/python/apache_beam/coders/coders.py

```diff
@@ -584,6 +594,9 @@ def __hash__(self):
     return hash(type(self)) + hash(self._value_coder)
 
 
+Coder.register_structured_urn(common_urns.coders.NULLABLE.urn, NullableCoder)
```

Comment:
Do we have tests to make sure that coders of the three languages are compatible ? For example, that you can encode a value of type "NullableCoder(Other StandardCoder)" from SDK1 and  decode from SDK2 and vice versa for all SDK combinations ?

@TheNeuralBit might know the answer.

Reply:
Yes. We have a yaml file that contains test cases that all three sdks read and validate. org/apache/beam/model/fnexecution/v1/standard_coders.yaml